vm/rust/src/lib.rs

```diff
@@ -55,22 +56,64 @@ extern "C" {
     fn JunoAppendActualFee(reader_handle: usize, ptr: *const c_uchar);
 }
 
-const N_STEPS_FEE_WEIGHT: f64 = 0.005;
+const GLOBAL_CONTRACT_CACHE_SIZE: usize= 100; // Todo ? default used to set this to 100.
+
+
+lazy_static! {
+    pub static ref FEE_TOKEN_ADDRESSES: FeeTokenAddresses = {
+        let eth_fee_token_address = StarkHash::try_from("0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7")
+            .and_then(|hash| ContractAddress::try_from(hash))
+            .unwrap();
+
+        let strk_fee_token_address = StarkHash::try_from("0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d")
+            .and_then(|hash| ContractAddress::try_from(hash))
+            .unwrap();
+
+        FeeTokenAddresses {
+            eth_fee_token_address,
+            strk_fee_token_address,
+        }
+    };
+}
+
+lazy_static! {
+    static ref VERSIONED_CONSTANTS: HashMap<String, VersionedConstants> = {
+        let mut m = HashMap::new();
+        m.insert("0.13.0".to_string(), serde_json::from_str(include_str!("../versioned_constants_13_0.json")).unwrap());
+        m.insert("".to_string(), serde_json::from_str(include_str!("../versioned_constants_13_0.json")).unwrap()); // use this by default
```

Comment:
why are they the same file?

Reply:
https://github.com/eqlabs/pathfinder/blob/8a983683e1b01cc579d5339012ef23c86b49f0fc/crates/executor/src/execution_state.rs#L50C21-L50C39

Reply:
I updated the code to use https://github.com/eqlabs/pathfinder/blob/main/crates/executor/src/execution_state.rs#L24